test/pom.xml

```diff
@@ -61,11 +61,23 @@
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>openshift-client</artifactId>
```

Comment:
I think we should have 2 clients in ST, the first one for OpenShift and the second one for Kubernetes because of differences in configuration. For example, OpenShift has different default namespace than Kubernetes etc...

Reply:
In that case, `Oc.java` should be refactored or removed?  @tombentley  WDYT about this?

Reply:
The `openshift-client` artifact provides the `OpenShiftClient`. You'll need that if you want to use OpenShift specific APIs like DeploymentConfig, BuildConfig etc.

The `Oc` class what we have is just a wrapper around the CLI `oc` tool. TBH I don't know how much of the functionality of `Oc` has been replaced by using the fabric8 client(s) at this point. If all the functionality could be done via `OpenShiftClient` then it would certainly be possible to remove it.